practice.md

```diff
@@ -49,6 +50,13 @@ You can then initialize a new 18F repository's license information with:
 
 It's also recommended to copy and paste [this paragraph for the end of your README](https://github.com/18F/open-source-policy/blob/master/README.md#public-domain) that sums up what's going on.
 
+### Accepting contributions from the public
+
+Any 18F project can accept open source contributions from the public. Contributors must agree to release their contributions into the international public domain.
```

Comment:
IANYL, but as a derivative work, it'd be GPL as would any contributions. Government should release into public domain or under least restrictive license legally allowed where bound by upstream projects. Technically, technically, if I were to take out one method, and that one method had no WordPress functions or API calls, I'd have an argument for using it as public domain. See generally http://ben.balter.com/2014/10/08/open-source-licensing-for-government-attorneys/ and http://ben.balter.com/2012/07/26/government-release-of-source-code-public-domain-or-open-source/.